racecar_simulator:
  ros__parameters:
    # General parameters
    simulator_frequency: 1000.0
    pub_frequency: 40.0
    friction_coefficient: 0.8
    scan_beams: 1080
    scan_field_of_view: 6.71238898038469  # 2 * M_PI * 0.75
    scan_std_dev: 0.01
    map_free_threshold: 0.2

    # Car 0 parameters
    drive_topic0: "ackermann_cmd0"
    state_topic0: "state0"
    scan_topic0: "scan0"
    mass0: 3.5
    l_r0: 0.17145
    l_f0: 0.17145
    I_z0: 0.04712
    B_f0: 1.5
    C_f0: 1.5
    D_f0: 30.0
    B_r0: 1.5
    C_r0: 1.5
    D_r0: 30.0
    steer_max0: 4.0
    steer_vel_max0: 4.0
    speed_max0: 10.0
    accel_max0: 40.0
    decel_max0: 40.0
    jerk_max0: 100.0

    # Car 1 parameters
    drive_topic1: "ackermann_cmd1"
    state_topic1: "state1"
    scan_topic1: "scan1"
    mass1: 3.5
    l_r1: 0.17145
    l_f1: 0.17145
    I_z1: 0.04712
    B_f1: 1.5
    C_f1: 1.5
    D_f1: 30.0
    B_r1: 1.5
    C_r1: 1.5
    D_r1: 30.0
    steer_max1: 0.4
    steer_vel_max1: 4.0
    speed_max1: 10.0
    accel_max1: 40.0
    decel_max1: 40.0
    jerk_max1: 100.0
